src/tests/admin/conferenceRoom/ForARoomDisabledTheTabletDoesNotScheduleMeetings.java

```diff
@@ -0,0 +1,62 @@
+package tests.admin.conferenceRoom;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SearchPage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC:24 Verify that when a room status is disabled, the tablet does not schedule meetings
+ * @author Juan Carlos Guevara
+ */
+public class ForARoomDisabledTheTabletDoesNotScheduleMeetings {
+	ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	List<Map<String, String>> testData = excelReader.getMapValues("Resources");
+	String roomName = testData.get(1).get("Room Name");
+
+	@Test(groups = {"FUNCTIONAL"})
+	public void testForARoomDisabledTheTabletDoesNotScheduleMeetings() {
+
+		//Disabling room in Admin 
+		HomeAdminPage homePage = new HomeAdminPage();
+		RoomsPage confRoomPage = homePage.clickConferenceRoomsLink();
+		if(confRoomPage.stateEnableDisableBtn(roomName)) {
+			confRoomPage.enableDisableIcon(roomName);
+		} else {
+
+			//Open tablet to see changes
+			HomeTabletPage homeTablet = new HomeTabletPage();
+			
+			/**
+			 * if this test case runs first this two lines should be without comments
+			 * SettingsPage settingsPage = homeTablet.clickSettingsBtn();
+			 * homeTablet = settingsPage.selectRoom(roomName);
+			 */
```

Comment:
This comment should not exist because you are making the test case depend on others